mkdocs/commands/gh_deploy.py

```diff
@@ -126,9 +127,15 @@ def gh_deploy(config, message=None, force=False, ignore_version=False):
         if host is None:
             # This could be a GitHub Enterprise deployment.
             log.info('Your documentation should be available shortly.')
+        if page_type == 'user' or page_type == "org":
+            username, repo = path.split('/', 1)
```

Comment:
A bunch of tests at `tests.gh_deploy_tests` are failing because `path` can be `None` here. You need to account for that.